.make/check_gopath.go

```diff
@@ -11,36 +11,45 @@ import (
 	"strings"
 )
 
+func isModuleFlagOn() bool {
+	var moduleFlag = os.Getenv("GO111MODULE")
+	if moduleFlag == "on"  {
```

Comment:
Please use strconv for parsing a boolean number:

```go
package main

import (
	"fmt"
	"strconv"
)

func main() {
	fmt.Println(strconv.ParseBool("true"))
	fmt.Println(strconv.ParseBool("1"))
	fmt.Println(strconv.ParseBool("false"))
	fmt.Println(strconv.ParseBool("0"))
}
```

Reply:
@kwk we can't apply stroconv.ParseBool operation on string `on`. Do you think there is other option ? considering go expects GO111MODULE variable value to be `on`(string)